install.sh

```diff
@@ -88,20 +85,42 @@ else
     initResponse=$(vault_i operator init -key-shares=1 -key-threshold=1)
     unsealKey=$(grep -C 1 "Unseal Key" <<< "$initResponse" | cut -d : -f 2 | xargs)
     rootToken=$(grep -C 1 "Root Token" <<< "$initResponse" | cut -d : -f 2 | xargs)
+    docker secret rm "${stackname}_vault_unseal"
+    echo -e "$unsealKey" | docker secret create "${stackname}_vault_unseal" - 
     vault_i operator unseal $unsealKey
     vault_i login $rootToken
+
+    # Configure root CA
```

Comment:
I'm sure that as this project grows, the install script will also grow with it. It might be a good idea to break the script out into separate methods or even better, separate scripts if you can.